src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
I see three options:

1.  Current, different illuminance output names for SplitFlux vs DElight - if you switch methods, then the available output variable names change.  That could be confusing.

2.  Generate both sets for both methods.  This would let the old variable names pass through and the user can choose between the ones keyed off the control object vs keyed off the reference point name.  It's kind of like node-name variable vs component inlet temperature variables.  This could be confusing because the names are so similar.

3.  Choose one or the other and use it for both methods.